docs/topics/js/js-overview.md

```diff
@@ -59,6 +59,61 @@ at the expense of type safety.
 Kotlin/JS is also compatible with the most common module systems: UMD, CommonJS, and AMD. Being able to [produce and consume modules](js-modules.md)
 means that you can interact with the JavaScript ecosystem in a structured manner.
 
+## Kotlin/JS frameworks
+
+Modern web development benefits significantly from frameworks that simplify building web applications.
+Here are examples of popular web frameworks for Kotlin/JS written by different authors:
+
+### KVision
+
+_KVision_ is an object-oriented web framework that makes it possible to write applications in Kotlin/JS without having to
+write HTML or CSS. Instead, it provides ready-to-use components that can be used as building blocks for your application’s
+user interface. You can use both reactive and imperative programming models to build your frontend, use connectors for Ktor,
+Spring Boot, and other frameworks to integrate it with your server-side applications, and share code using Kotlin Multiplatform.
+
+Visit [https://kvision.io](https://kvision.io) for documentation, tutorials, and examples.
+
+For updates and discussions about the framework, join [#kvision](https://kotlinlang.slack.com/messages/kvision) and
+[#javascript](https://kotlinlang.slack.com/archives/C0B8L3U69) channels in the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
+
+### fritz2
+
+_fritz2_ is a standalone framework for building reactive web user interfaces. It provides its own type-safe DSL for building
+and rendering HTML elements, and it makes use of Kotlin’s coroutines and flows to express components and their data bindings.
+It provides state management, validation, routing, and more out of the box, and integrates with Kotlin Multiplatform projects.
+
+Visit [https://www.fritz2.dev](https://www.fritz2.dev) for documentation, tutorials, and examples.
+
+Join the [#fritz2](https://kotlinlang.slack.com/messages/fritz2) and [#javascript](https://kotlinlang.slack.com/archives/C0B8L3U69)
+channels in the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up)to stay up to date with the latest
+news about this framework.
+
+### Doodle
+
+_Doodle_ is a vector-based UI framework for Kotlin/JS. Apps built using it do not rely on DOM, CSS, or Javascript. Instead,
+they use the browser’s graphics capabilities to draw user interfaces, giving you precise control over the rendering of
```

Comment:
Again, why don't we use the following order: Application use this, and don't rely on that?